starport/services/serve/rly/relayer.go

```diff
@@ -0,0 +1,82 @@
+package rly
+
+import "fmt"
+
+type GlobalConfig struct {
+	Timeout       string `yaml:"timeout" json:"timeout"`
+	LiteCacheSize int    `yaml:"lite-cache-size" json:"lite-cache-size"`
+}
+
+type Chains []*Chain
+
+type Chain struct {
```

Comment:
I think we should describe the fields in comments, some terms can be hard to understand to someone not initiated with IBC

Reply:
I agree! To make it easier for us (and to avoid duplication), we can link to the relevant specs:

For channels, for example: https://github.com/cosmos/ics/tree/master/spec/ics-004-channel-and-packet-semantics

Reply:
done by 9f4ad8c